src/db.py

```diff
@@ -239,6 +239,14 @@ def create_submission(self, url_name, long_name, parent_project_url, owner):
 
         return submission_db.insert_one(submission_obj)
 
+    def delete_submission(self, long_name, owner, gitolite_url):
+        submission_db = self.mongo.gitsubmit.submissions
+        submission_obj = submission_db.find({"long_name": long_name, "owner": owner, "gitolite_url": gitolite_url})
+        
+        if submission_obj is not None:
+            submission_db.drop(submission_db)
```

Comment:
this line would delete the entire database. I'm pretty sure you meant `submission_db.remove(submission_obj["_id"])`